lib/OpenQA/Task/Screenshot/Scan.pm

```diff
@@ -25,8 +25,7 @@ use OpenQA::Utils qw(log_debug log_warning log_fatal);
 
 sub register {
     my ($self, $app) = @_;
-    $app->minion->add_task(scan_images       => sub { _scan_images($app, @_) });
-    $app->minion->add_task(scan_images_links => sub { _scan_images_links($app, @_) });
```

Comment:
I'm wondering when this task is executed. But it seems we have at least tests for it which needed to be removed as well.

Reply:
scan_images_links was triggered from dbicdh/_common/upgrade/47-48/04-scan-images-links.pl - removed in 334f53ac011ba2bda98a9d8db6be7e36cd284491

Didn't bother to check scan_images